tests/unit/unit1652.c

```diff
@@ -91,7 +91,7 @@ fail_unless(strcmp(result, "Simple Test 42 testing 43\n") == 0,
 
 /* Variations of empty strings */
 Curl_infof(data, "");
-fail_unless(strlen(result) == 0, "Empty string");
+fail_unless(!*result, "Empty string");
```

Comment:
I object to this change. This is testcode with no performance implications, this code should be kept as simple to read as possible which IMO is keeping `strlen()`.

Reply:
Done